app/models/user.rb

```diff
@@ -82,12 +82,18 @@ class User
   # email presence validated through Devise
   validates :name, presence: true
   validates :role, presence: true
-  validate :secure_password
+  validate :secure_password, if: :updating_password?
+  # i18n-tasks-use t('errors.messages.password.password_strength')
```

Comment:
Why is this commented out?

Reply:
it tells our i18n linter that we do use the value